sw/device/tests/BUILD

```
@@ -3272,6 +3279,36 @@ opentitan_test(
     ],
 )
 
+opentitan_test(
+    name = "spi_device_ujson_ottf_test",
+    srcs = ["spi_device_ujson_ottf_test.c"],
```

Can we rename this something like:
```suggestion
    name = "spi_device_ujson_console_test",
    srcs = ["spi_device_ujson_console_test.c"],
```

and likewise the `test_harness` below? 

The current name is confusing since there is also a `spi_device_ottf_console_test` that just tests the SPI device OTTF console, without sending ujson data payloads I believe.

---

Sure, done.